packages/cli-kit/src/private/node/ui/components/Prompts/GitDiff.tsx

```diff
@@ -19,7 +21,7 @@ export interface GitDiffProps {
  *   unchanged line
  * + added line
  */
-const GitDiff: FunctionComponent<GitDiffProps> = ({baselineContent, updatedContent}): JSX.Element => {
+const GitDiff: FunctionComponent<GitDiffProps> = ({gitDiff: {baselineContent, updatedContent}}): JSX.Element => {
```

Comment:
Can you please elaborate a bit on the value of nesting these properties? I'm not finding the rationale for the change obvious, but I'm sure there's a good reason.